README.md

```diff
@@ -838,6 +897,13 @@ prefix `--terragrunt-`. The currently available options are:
 * `--terragrunt-source-update`: Delete the contents of the temporary folder before downloading Terraform source code
   into it.
 
+## The old .terragrunt configuration file
+
+Older versions of Terragrunt used to read their configuration from a `.terragrunt` file instead of `terraform.tfvars`.
+This file format is now **deprecated**, although for backwards compatibility, Terragrunt will continue to support it
+for the near future. The configuration in a `.terragrunt` file is identical to that of the `terraform.tfvars` file 
+except it does *not* need to be wrapped with a `terragrunt = { ... }` block.
+
```

Comment:
Agreed these docs are getting longer. Clean up for that definitely belongs in a separate PR.